models/particle_net.py

```diff
@@ -150,7 +163,8 @@ def __init__(self, K, channels, with_bn=True, activation='relu', pooling='mean',
 
     def call(self, points, features):
         d = self.batch_distance_matrix_general(points, points)
-        _, indicies = tf.nn.top_k(-d, k=self.K + 1)
+        k = self.K if tf.shape(features)[1] >= 16 else tf.shape(features)[1]
+        _, indicies = tf.nn.top_k(-d, k=k + 1)
         indicies = indicies[:,:,1:]
 
```

Comment:
I would assume that `self.K` should be changed to `k` everywhere in the `call()` method of `EdgeConv`, shouldn't it? so basically also in lines 171-172:
 
https://github.com/ofivite/tatoo/blob/15b5d8767fb7a087d36eb4dd643d3037fc6bcc21/models/particle_net.py#L171-L172